homeassistant/components/google_cloud/const.py

```diff
@@ -18,3 +19,166 @@
 CONF_GAIN = "gain"
 CONF_PROFILES = "profiles"
 CONF_TEXT_TYPE = "text_type"
+
+# STT constants
+CONF_STT_MODEL = "stt_model"
+
+DEFAULT_STT_MODEL = "latest_short"
+
+# https://cloud.google.com/speech-to-text/docs/transcription-model
+SUPPORTED_STT_MODELS = [
+    "latest_long",
+    "latest_short",
+    "telephony",
+    "telephony_short",
+    "medical_dictation",
+    "medical_conversation",
+    "command_and_search",
+    "default",
+    "phone_call",
+    "video",
+]
+
+# https://cloud.google.com/speech-to-text/docs/speech-to-text-supported-languages
+STT_LANGUAGES = [
+    "af-ZA",
```

Comment:
We don't have this list in the lib probably?

Reply:
Unfortunately not. They are only mentioned in a comment at https://github.com/googleapis/google-cloud-python/blob/957778b3fd25529bbb3f85dac3d9922a50fdaad9/packages/google-cloud-speech/google/cloud/speech_v1/types/cloud_speech.py#L424-L478 and https://github.com/googleapis/google-cloud-python/blob/957778b3fd25529bbb3f85dac3d9922a50fdaad9/packages/google-cloud-speech/google/cloud/speech_v1/types/cloud_speech.py#L323C9-L328